src/EFCoreSecondLevelCacheInterceptor/EFInfinispanCacheServiceProvider.cs

```diff
@@ -12,13 +17,15 @@ public class EFInfinispanCacheServiceProvider : IEFCacheServiceProvider
 {
     InfinispanDG dg = new InfinispanDG();
     Cache<EFCacheKey, EFCachedData> cache;
+    Cache<string, HashSet<EFCacheKey>> depCache;
     public EFInfinispanCacheServiceProvider()
     {
         // Use a non-authenticated non-encrypted cluster;
         dg.AddHost("127.0.0.1", 11222);
```

Comment:
Is there any specific reason to use non-authenticated cluster ?

Reply:
just to keep things a simple as possible for a POC.
All the configuration part is actually missing currently and needs a plan. Authentication will be there